.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
I am not sure. Do we use indent_size of 4 at the moment?

Reply:
We use ktlint for kotlin files and the indentation is not 2 but partially 3 and 4.